src/Cardano/Db/Database.hs

```diff
@@ -93,7 +94,9 @@ runActions trce env plugin actions = do
             runRollbacks trce plugin pt
             dbAction Continue ys
         (ys, zs) -> do
-          insertBlockList trce env plugin ys
+          insertBlockList trce env
+            undefined -- ??
```

Comment:
I don't know what to pass here. This is `LedgerStateVar`, which isn't even exposed, see https://github.com/input-output-hk/cardano-db-sync/issues/373